conda-forge.yml

```diff
@@ -4,3 +4,8 @@ github:
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
+  osx_arm64: osx_64
+  win_64: win_64
```

Comment:
You don't need this here for windows, but you do need to delete
https://github.com/conda-forge/backtrace-feedstock/blob/70f9775450e285ceb6d27683ad452abb13b8e992/recipe/meta.yaml#L15